mappings/net/minecraft/tag/TagsHolder.mapping

```diff
@@ -0,0 +1,49 @@
+CLASS net/minecraft/class_6880 net/minecraft/tag/TagsHolder
+	METHOD method_40220 contains (Lnet/minecraft/class_6862;)Z
+		ARG 1 tagKey
+	METHOD method_40222 areRegistriesEqual (Lnet/minecraft/class_2378;)Z
+		ARG 1 registry
+	METHOD method_40223 createDirect (Ljava/lang/Object;)Lnet/minecraft/class_6880;
+		ARG 0 value
+	METHOD method_40224 testHolder (Ljava/util/function/Predicate;)Z
+		ARG 1 predicate
+	METHOD method_40225 isHeldBy (Lnet/minecraft/class_5321;)Z
+		ARG 1 holderRegistryKey
+	METHOD method_40226 areHolderIdsEqual (Lnet/minecraft/class_2960;)Z
+		ARG 1 id
+	METHOD method_40227 isBootstrapped ()Z
+	METHOD method_40228 stream ()Ljava/util/stream/Stream;
+	METHOD method_40229 toEither ()Lcom/mojang/datafixers/util/Either;
```

Comment:
Probably incorrect

Reply:
Hmm, I recall this looked like a holder of registry element than tag related... might need to look at it again later